tests/test_basic.py

```diff
@@ -414,11 +414,14 @@ def test_edit(runner, default_database):
     todo = FileTodo()
     todo.list = next(default_database.lists())
     todo.summary = 'Eat paint'
+    todo.location = 'Boston'
```

Comment:
Please write a separate test for this, rather than injecting a location into a tests that already tests for something else. On that starts of similar to `test_show_location` would suffice really (eg, use `create`, and the use `runner` to edit do the location`)